rosa_hcp/rosa-hcp-shared-vpc-config.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="rosa-hcp-shared-vpc-config"]
+= Configuring a shared VPC for ROSA with HCP clusters
+:context: rosa-shared-vpc-config
+
+toc::[]
+
+You can create {hcp-title-first} clusters in shared,centrally-managed AWS virtual private clouds (VPCs).
+
+[NOTE]
+====
+This process requires *two separate* AWS accounts that belong to the same AWS organization. One account functions as the VPC-owning AWS account (*VPC Owner*), while the other account creates the cluster in the cluster-creating AWS account (*Cluster Creator*).
+====
+
+image::522-shared-vpc-overview.png[]
+
+* The hosted zones can be created in either the centrally-managed VPC account or in the workload account in which the cluster is deployed.
+
+.Prerequisites for the *VPC Owner*
+* You have an AWS account with the proper permissions to create roles and share resources.
+* The *Cluster Creator's* AWS account is separate from the *VPC Owner's* AWS account.
```

Comment:
must the accounts be separate? would this work with multiple VPCs in the same account?

Reply:
Not sure if I follow, why would a customer attempt to share a vpc with the same account that created the vpc?

Reply:
This will not work with multiple VPCs in the same account. I think we should remove this line.

Reply:
Why remove it @michaelryanmcneill ? It is the vpc resource that is mentioned it needs to be in a separate account shared to the cluster owner account. Why wouldn't it work with multiple vpcs in the same account?